dapps/marketplace/src/pages/listings/_Search.js

```diff
@@ -7,49 +7,145 @@ import { fbt } from 'fbt-runtime'
 import { getStateFromQuery } from './_filters'
 
 import withConfig from 'hoc/withConfig'
+import withIsMobile from 'hoc/withIsMobile'
 
 class Search extends Component {
```

Comment:
wondering if we can turn this into a functional component. No worries if not... in general trying to take the opportunity to convert to functional when working on those parts of the code, but only if it's not a big pain.

Reply:
Will give it a try